engine/crates/parser-openapi/src/graph/operations.rs

```diff
@@ -238,7 +238,8 @@ impl DebugNode for Operation {
 }
 
 fn is_json(content_type: &str) -> bool {
-    content_type == "application/json"
+    // This covers `application/json` and the jsonAPI style `application/vnd.whatever+json`
+    content_type.ends_with("json")
```

Comment:
Do we want to check for `/json` or `+json` specifically to reduce edge cases?

Reply:
I'd rather be lax than strict on this tbh.

Unless you have specific examples of content types that end in `json` but are actually not json?